instrumentation/azure-functions/src/main/java/io/opentelemetry/instrumentation/auto/azure/functions/AzureFunctionsInstrumentationHelper.java

```diff
@@ -0,0 +1,50 @@
+package io.opentelemetry.instrumentation.auto.azure.functions;
+
+import com.google.common.base.Strings;
+import io.opentelemetry.instrumentation.api.aiconnectionstring.AiConnectionString;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AzureFunctionsInstrumentationHelper {
```

Comment:
I suggest restructuring this class a bit to make it testable, and writing some unit tests, since the logic is a bit complex

Reply:
will do